### PR TITLE
ENH:Fixes to get() method following changes in metadataclient

### PR DESCRIPTION
**Pull request by arkilic**
Cleanup to follow

### CODE REVIEW

File: metadataservice/server/engine.py [hunk 1]
```diff
@@ -59,7 +57,12 @@ def get(self):
             utils.report_error(400,
                                'A query string must be provided')
         docs_gen = func(**query)
-        utils.transmit_list(self, list(docs_gen))
+        if isinstance(docs_gen, (doct.Document, list)):
```
Comment (tacaswell) [inline metadataservice/server/engine.py]: What returns a single document?

Comment (arkilic) [inline metadataservice/server/engine.py]: *_by_uid

Comment (arkilic) [inline metadataservice/server/engine.py]: stop_by_start

Comment (tacaswell) [inline metadataservice/server/engine.py]: Ah, that makes sense (suspect I even have advocated for that API at the time).

**Pull request merged**